HPL2/core/include/BuildID_HPL2_0.h

```diff
@@ -1,31 +1,12 @@
-/*
```

Comment:
You shouldn't remove the copyright notice.

Reply:
This file is auto generated... I'm not sure why the copyright notice went away :/